crates/cubecl-runtime/src/memory_management/memory_manage.rs

```diff
@@ -116,11 +112,11 @@ impl<Storage: ComputeStorage> MemoryManagement<Storage> {
             MemoryConfiguration::SubSlices => {
                 // Round chunk size to be aligned.
                 let memory_alignment = properties.alignment;
-                let max_page = round_up_to_multiple(properties.max_page_size, memory_alignment);
+                let max_page = properties.max_page_size;
```

Comment:
Doesn't `max_page` need to be aligned?

Reply:
Doesn't always seem like it, I'm guessing because wgpu clamps to 4gb to align things. In any case would like to allow aligning to higher align than wgpu requires